diff -p -up contacts-0.11/src/contacts-callbacks-ui.c~ contacts-0.11/src/contacts-callbacks-ui.c
--- src/contacts-callbacks-ui.c~	2009-07-14 16:00:54.000000000 +0200
+++ src/contacts-callbacks-ui.c	2009-11-10 15:06:57.000000000 +0100
@@ -221,7 +221,7 @@ contacts_delete_cb (GtkWidget *source, C
 	dialog = gtk_message_dialog_new (GTK_WINDOW (main_window),
 					 0, GTK_MESSAGE_QUESTION,
 					 GTK_BUTTONS_CANCEL,
-					 message);
+					 "%s", message);
 	gtk_dialog_add_buttons (GTK_DIALOG (dialog), GTK_STOCK_DELETE,
 		GTK_RESPONSE_YES, NULL);
 
